config/routes.rb

```diff
@@ -109,6 +109,10 @@
   resources :dashboard, only: [] do
     collection do
       get 'activity', action: :activity, as: :dashboard_activity
+      get 'update_collections', action: :update_collections, as: :update_collections
```

Comment:
These actions should be on their own controller (or maybe 3 separate controllers) specifically for analytics.